docs/Admin.md

```diff
@@ -164,6 +164,16 @@ await admin.fetchTopicOffsets(topic)
 //   { partition: 2, offset: '32103', high: '32103', low: '518' },
 //   { partition: 3, offset: '28', high: '28', low: '0' },
 // ]
+
```

Comment:
I can't add a suggestion for this, as Github's markdown doesn't support it, but there should be a \``` here to close the previous code block. Currently the heading below is part of the code block.

Reply:
OK. Will add.